libcaf_net/test/net-test.cpp

```diff
@@ -82,7 +82,6 @@ ptrdiff_t mock_stream_transport::handle_input() {
     if (consumed < 0) {
       // Negative values indicate that the application encountered an
       // unrecoverable error.
-      up->abort(make_error(caf::sec::runtime_error, "consumed < 0"));
```

Comment:
I wanted to check weather removing this is OK, since the framing layer will call abort on it's own. This just makes calling it twice with different error codes, so testing is harder.